ui-kit/src/main/kotlin/org/p2p/uikit/components/UiKitSliderSolidButton.kt

```diff
@@ -93,6 +94,11 @@ class UiKitSliderSolidButton @JvmOverloads constructor(
         textViewActionTop.text = actionText
     }
 
+    fun setActionText(actionText: TextContainer) = with(binding) {
+        actionText.applyTo(textViewAction)
```

Comment:
can we make extensions to use this method like
textViewAction.bind(actionText) 
?

```
TextView.bind(container: TextContainer) { 
    actionText.applyTo(this)
}
```
to unify working with Cell models and Containers